package.json

```diff
@@ -67,6 +67,7 @@
     "acorn-jsx": "5.3.2",
     "autoprefixer": "10.4.14",
     "babel-core": "7.0.0-bridge.0",
+    "babel-jest": "29.5.0",
```

Comment:
Do we need this dependency? We have `@swc/jest` that transpiles module files, I've left a comment on how we can utilize it for `mjs` files!